automated install/basic-install.sh

```diff
@@ -413,6 +450,75 @@ package_manager_detect() {
     fi
 }
 
+build_dependency_package(){
+    # This function will build a package that contains all the dependencies needed for Pi-hole
+
+    # remove any leftover build directory that may exist
+    rm -rf /tmp/pihole-meta
+
+    # Create a fresh build directory
+    mkdir -p /tmp/pihole-meta
+    chmod 0755 /tmp/pihole-meta
+
+    if is_command apt-get; then
+        # move into the tmp directory
```

Comment:
Since we are using `/tmp` and don't need to have any persistence, what do you think about borrowing from the debug script and creating a randomly named directory to use? Store the path in a variable for use throughout the function. And remove the directory, if created, when the trap fires as well.

Reply:
Done.